app/components/content-container.tsx

```diff
@@ -20,7 +20,7 @@ export function ContentContainer({
     "items-center",
     "justify-center",
     "space-y-3",
```

Comment:
It really is large! And large, frankly, is pointless, even on massive comical monitors.